core/src/main/java/io/questdb/cutlass/line/http/LineHttpSender.java

```diff
@@ -286,18 +287,21 @@ private static boolean keepAliveDisabled(HttpClient.ResponseHeaders response) {
         return connectionHeader != null && Utf8s.equalsAscii("close", connectionHeader);
     }
 
-    private static long unitToNanos(ChronoUnit unit) {
+    private static long toMicros(long value, ChronoUnit unit) {
```

Comment:
Would be great to merge this method and `AbstractLineSender#toMicros()`. How about moving it to `Timestamps`?

Reply:
good idea